src/platform/packages/private/kbn-esql-editor/src/esql_editor.tsx

```diff
@@ -239,6 +271,42 @@ export const ESQLEditor = memo(function ESQLEditor({
     openTimePickerPopover();
   });
 
+  monaco.editor.registerCommand('esql.control.time_literal.create', async (...args) => {
```

Comment:
ℹ️ As the flow starts from the autocomplete we can easily guess that the variable will be an interval, field or value. We need the enum to ensure that we suggest correct options to the users.